metadata.json

```diff
@@ -21,7 +21,7 @@
       "version_requirement": ">= 6.4.0 < 8.0.0"
     },
     {
-      "name": "camptocamp/systemd",
+      "name": "puppet/systemd",
       "version_requirement": ">= 2.0.0 < 4.0.0"
```

Comment:
The fixtures has 4.2.0 so this version range needs to be updated to include the latest version released.

Reply:
@treydock the version limits have been updated